.cloudbuild/stage.yaml

```diff
@@ -1,43 +1,74 @@
-timeout: 2700s # set build timeout to 45 mins
+timeout: 2700s # Set build timeout to 45 mins
 steps:
-  - name: node:18
+  - id: 'install'
+    name: node:18
     entrypoint: npm
     args: ['ci']
 
-  - name: node:18
-    entrypoint: npm
-    args: ['run', 'production']
-    env:
-      - 'NODE_OPTIONS="--max_old_space_size=8192"'
+  - id: 'prepare'
+    name: node:18
+    entrypoint: bash
+    args:
+      - '-c'
+      - |
+        npm run stage:github determineDeploymentType &&
+        npm run stage:github announceDeploymentStart
 
-  - name: node:18
-    entrypoint: npm
+  - id: 'build'
+    name: node:18
     env:
-      - 'BRANCH_NAME=$BRANCH_NAME'
-      - 'PR_NUMBER=$_PR_NUMBER'
-    secretEnv: ['GITHUB_TOKEN']
-    args: ['run', 'stage:github', 'determineDeploymentType']
+      - 'CI=true'
+      - 'NODE_OPTIONS="--max_old_space_size=8192"'
+      - 'ELEVENTY_IGNORE_NACL=true'
```

Comment:
We didn't ignore this previously, did we?

Reply:
also shoukld not influence prod build i think, and you run "npm run production" later on in gulp?

Reply:
It does by setting `CI=true` (which is weird, but here we are). That's how it currently is for Netlify, and it takes ~1m from the build (less files to build, less files to upload).

I am a little bit torn about it: I think it's good to do this, to not have people (rightfully) complain that the new previews are slower to build (which they are not measured no a per page basis) on the other hand I actually liked having **full** builds. What's your stance?

Reply:
we can ignore nacl for PRs because they are fully automated content, which is unlikely to get influenced by random PR.